data/json/items/ammo.json

```diff
@@ -298,8 +298,9 @@
     "color": "dark_gray",
     "description": "Plastic jacketed copper cable of the type used in small electronics.",
     "material": [ "plastic", "copper" ],
-    "volume": "750 ml",
-    "weight": "2 g",
+    "//": "Calculated off of 12 AWG, at 10cm length, using http://www.mtlexs.com/technical-specifications-details/40/weight-of-copper-wire.",
+    "volume": "437 ml",
+    "weight": "984 g",
```

Comment:
The table at the link lists diameters, not radii, so the correct volume for 200 charges of 10 cm wire is

π / 4 * diameter<sup>2</sup> * length = π / 4 * (2.64mm)<sup>2</sup> * 200 * 10cm ≈ 109ml

The "weight" value represents weight per charge instead of per default stack, so the value should be 4.92 g or approximately 5 g.

Reply:
How could I have missed that? I knew the numbers seemed off!

Reply:
It is mentioned once under "Generic items", in the comment for "weight".

```
"weight": "350 g",                           // Weight, weight in grams, mg and kg can be used - "50 mg", "5 g" or "5 kg". For stackable items (ammo, comestibles) this is the weight per charge.
```

Reply:
Ah, I didn't notice that.